nemo/collections/asr/parts/numba/rnnt_loss/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018-2019, Mingkun Huang
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is it necessary to have this license here?

Reply:
Yes, here its needed since the 2 functions and the class is ported from that codebase.

Reply:
But this is the init file which just contains the imports.

Reply:
Removed